dbms/src/Storages/tests/gtest_SplitTokenExtractor.cpp

```diff
@@ -0,0 +1,174 @@
+#include <Storages/MergeTree/MergeTreeIndexFullText.h>
+
+#include <Common/PODArray_fwd.h>
+#include <Common/PODArray.h>
+
+#include <gtest/gtest.h>
+
+#include <vector>
+#include <string_view>
+#include <string>
+#include <memory>
+
+namespace
+{
+using namespace DB;
+}
+
+struct SplitTokenExtractorTestCase
+{
+    const char * description;
```

Comment:
nit: please, use std::string_view as operator<< is not going to call strlen each time

Reply:
done